docs/visual-basic/programming-guide/concepts/expression-trees/debugview-syntax.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Syntax used by DebugView property (Visual Basic)
+description: Describes the special syntax used by the DebugView property to produce a string representation of expression trees
+author: zspitz
+ms.author: wiwagn
+ms.date: 05/22/2019
+ms.topic: reference
+helpviewer_keywords: expression-trees, debugview
+---
+
+# `DebugView` syntax 
+
+The `DebugView` property (available only when debugging) provides a string rendering of expression trees. Most of the syntax is fairly straightforward to understand; the special cases are described in the following sections.
+
+Each example is followed by a comment block containing the `DebugView`. 
+
+## ParameterExpression
+
+<xref:System.Linq.Expressions.ParameterExpression> variable names are displayed with a "$" symbol at the beginning.
```

Comment:
```suggestion
<xref:System.Linq.Expressions.ParameterExpression?displayProperty=nameWithType> variable names are displayed with a "$" symbol at the beginning.
```

Same comment as the C# version. (And yes, I do realize I missed this in reviewing the earlier PR that added this information. That's my fault. 😞 )

Reply:
No problem; we're all human (except for [this user](https://xkcd.com/1530/)). 😊

Reply:
Fixed.